app/src/main/java/org/eyeseetea/malariacare/data/database/model/MediaDB.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2015.
+ *
+ * This file is part of QA App.
+ *
+ *  Health Network QIS App is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Health Network QIS App is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Foobar.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.eyeseetea.malariacare.data.database.model;
+
+import com.raizlabs.android.dbflow.annotation.Column;
+import com.raizlabs.android.dbflow.annotation.PrimaryKey;
+import com.raizlabs.android.dbflow.annotation.Table;
+import com.raizlabs.android.dbflow.sql.language.Select;
+import com.raizlabs.android.dbflow.structure.BaseModel;
+
+import org.eyeseetea.malariacare.data.database.AppDatabase;
+import org.eyeseetea.malariacare.utils.Constants;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Table(database = AppDatabase.class, name = "Media")
+public class MediaDB extends BaseModel {
+
+    /**
+     * Null media value to express that a question has NO media without using querys
+     */
+    private static MediaDB noMedia = new MediaDB(Constants.NO_MEDIA_ID, null, null);
+
+    @Column
+    @PrimaryKey(autoincrement = true)
+    long id_media;
+
+    @Column
+    int media_type;
+
+    @Column
+    String resource_url;
+
+    @Column
+    Long id_question_fk;
+
+    @Column
+    String filename;
+
+
+    /**
+     * Reference to the question
+     */
+    QuestionDB question;
+
+    public MediaDB() {
+    }
+
+    public MediaDB(int media_type, String resource_url, QuestionDB question) {
+        this.media_type = media_type;
+        this.resource_url = resource_url;
+        this.filename = null;
+        this.setQuestion(question);
+    }
+
+    public QuestionDB getQuestion(){
```

Comment:
We are going to move all necessary queries to MediaRepository and MediaDB only should have columns for table